recipes/ira/recipe.yaml

```
@@ -0,0 +1,76 @@
+context:
+  version: "2.2.0"
+  build_num: 0
+
+package:
+  name: ira
+  version: ${{ version }}
+
+source:
+  url: https://github.com/mammasmias/IterativeRotationsAssignments/archive/refs/tags/IRA_v${{ version }}.tar.gz
+  sha256: 06d906512c6745a95c6eb6c7b5f2e2de95a037d45573e65b2fc4c7db1318d562
+
+build:
+  number: ${{ build_num }}
+  string: f${{ hash }}_${{ build_num }}
+  skip:
+    - win
+  script: build.sh # Switched to build.sh
+
+requirements:
+  build:
+    # Compilers are the main build dependency
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('fortran') }}
+    - ${{ stdlib('c') }}
+    - if: osx # From eon recipe
+      then: gfortran_impl_osx-64 >14
+
+  host:
+    # Build tools and runtime dependencies
+    - python
+    - pip
+    - setuptools # Required by setup.py
+    - cmake      # Required by setup.py
+    - ninja
+    - numpy
+    - libblas
+    - liblapack
+    - ${{ stdlib('c') }} # Handles libgcc/libstdcxx dependencies
+    - if: osx # From eon recipe
+      then: llvm-openmp
+
+  run:
+    # Runtime dependencies
+    - python
+    - numpy
+    - if: osx
+      then: llvm-openmp
```

```suggestion
requirements:
  build:
    - ${{ compiler('c') }}
    - ${{ compiler('cxx') }}
    - ${{ compiler('fortran') }}
    - ${{ stdlib('c') }}
    - cmake
    - ninja
    - if: (build_platform != target_platform)
      then:
        - cross-python_${{ target_platform }}
        - numpy
        - python
  host:
    - python
    - pip
    - setuptools
    - numpy
    - libblas
    - liblapack
  run:
    - python
    - numpy
```
Some changes are needed for the dependencies:
- `cmake` and `ninja` should be in build, not host
- Cross compilation dependencies were added since this is native Python package
- OpenMP is not used by the recipe, so `llvm-openmp` is not needed.
- Let's see if defaults to `gfortran` needs to be specified at 14 any more.  If it does we can handle this in `variants.yaml`.